### ISSUE TITLE
new Term: horizontal clavicles

### ISSUE THREAD

**Issue by CorneliusDreyling** [obophenotype/human-phenotype-ontology#10801]
**Preferred term label:**
horizontal clavicles

**Synonyms**
horizontal positioning of the clavicles

**Definition (free text, please give PubMed ID)**
Phenotype that manifests itself in neuromuscular diseases. Due to muscle weakness of the periscapular muscles, specifically weakness of the lower trapezius muscle the clavicles are positioned almost paralel to the ground. (https://doi.org/10.1007/s13311-018-00675-3)

**Parent term (use hpo.jax.org/app)**
Abnormal clavicle morphology HP:0000889

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
FSHD Orphanet Code: 269

**Your nano-attribution (ORCID)**
0009-0004-9942-3886

**Comment by pnrobinson** [comment]
@CorneliusDreyling 
The clavicles are said to be horizontal physiologically (https://www.physio-pedia.com/Clavicle), and their actual position varies.
I wonder if this is the same as [Down-sloping shoulders HP:0200021](https://hpo.jax.org/browse/term/HP:0200021) ?

**Comment by pnrobinson** [comment]
@CorneliusDreyling I think that the description of clavicles as "horizontal" made sense in the context of a specific discussion but this is not a general abnormality and would be confusing as an HPO term. I will close this for now, but we can reopen it if needed.
